repondeur/zam_repondeur/templates/reponse_edit.html

```diff
@@ -0,0 +1,120 @@
+{% extends "_base.html" %}
+
+{% from "_macros.html" import lecture_title, amendement_title %}
+
+
+{% block header %}
+    <style type="text/css">
+        h4 {
+            margin-bottom: 0.75rem;
+        }
+        .auteur, .dispositif, .objet {
+            margin-bottom: 1.5rem;
+        }
+        .auteur {
+            font-style: italic;
+        }
+        label {
+            font-weight: 500;
+        }
+        textarea {
+            height: 30vh;
+        }
+        div.mce-fullscreen {
+            z-index: 1030;
+        }
+    </style>
+    <script src="{{ request.static_url('zam_repondeur:static/tinymce/tinymce.min.js') }}"></script>
+    <script type="text/javascript">
+        tinymce.init({
+            selector: "textarea",
+            language: "fr_FR",
+            menubar: false,
+            plugins: "lists fullscreen",
+            toolbar: "undo redo | bold italic underline strikethrough removeformat | bullist numlist | cut copy paste | fullscreen",
```

Comment:
I would let `bold`, `italic` and `fullscreen` only for now. And then add more options only if real blockers. When I see how current answers are full of bold I'm afraid we will have meaningless styles otherwise…

Reply:
I'm removing `underline` and `strikethrough` → 4e250dd9d0dd661dfe37ae08e2fac5028b28fe6f

OK to keep the `bullist` and `numlist`?